src/App.js

```diff
@@ -1,29 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
```

Comment:
Please make sure that the `App.js` component is class-based. As much as it is better to use functional components and you will later refactor your code. The point of the curriculum is to learn how to use both class-based and functional components. For now, let's use class-based components until further notice.